common/views/components/PrismicImage/PrismicImage.tsx

```diff
@@ -4,11 +4,16 @@ import styled from 'styled-components';
 import { Breakpoint, sizes as breakpointSizes } from '../../themes/config';
 import { ImageType } from '../../../model/image';
 
+// Note: for some reason passing a boolean to desaturate causes a warning;
+// see the commit message for the full stack trace.
+//
+// Since this wrapper is only used in this file, we use the slightly clunkier
+// 1 = true / 0 = false to silence the warning.
```

Comment:
So strange! Is it because it could be undefined?

Reply:
It might be that a [transient prop](https://styled-components.com/docs/api#transient-props) was needed? I've only encountered a need for it once or twice so not an expert, but next/image might be the one complaining about it and we only need it for styling purposes...